File: app/src/main/java/upb/pdp/models/Persona.java
package upb.pdp.models;

public class Persona {


    private String nombre;

    private String telefono;
    private int edad;

    private String tipoDocumento;
    private String documento;

    @Override
    public String toString() {
        return String.format("Hola soy %s tengo %d mi documento es %s %s y mi teléfono es %s", this.nombre, this.edad, this.tipoDocumento, this.documento, this.telefono);
    }

    public Persona(String nombre, String telefono, int edad, String tipoDocumento, String documento) {
        this.nombre = nombre;
        this.telefono = telefono;
        this.edad = edad;
        this.tipoDocumento = tipoDocumento;
        this.documento = documento;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getTelefono() {
        return telefono;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    public int getEdad() {
        return edad;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }

    public String getTipoDocumento() {
        return tipoDocumento;
    }

    public void setTipoDocumento(String tipoDocumento) {
        this.tipoDocumento = tipoDocumento;
    }

    public String getDocumento() {
        return documento;
    }

    public void setDocumento(String documento) {
        this.documento = documento;
    }
}

File: pdpapp/PDP/Models/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using PDP.Enums;

namespace PDP.Models
{
    public class Persona
    {
        public Persona(string nombre, uint edad, string telefono, TipoDocumento tipoDeDocumento, string documento)
        {
            Nombre = nombre;
            Edad = edad;
            Telefono = telefono;
            TipoDeDocumento = tipoDeDocumento;
            Documento = documento;
        }

        public String Nombre {  get; set; }
        public uint Edad { get; set; }
        public String Telefono {  get; set; }
        public TipoDocumento TipoDeDocumento { get; set; }
        public String Documento { get; set; }
        public String ColorDelCabello { get; set; }
        public String EstadoCivil {  get; set; }

        public void Saludar()
        {
            String saludo = String.Format("Hola me llamo {0} y tengo {1} años", 
                Nombre, 
                Edad);
            Console.WriteLine(saludo);
        }

        public void ImprimirDatos()
        {
            String datos = String.Format("Nombre: {0}\nEdad:{1}\nTelefono:{2}\nDocumento:{3} {4}", 
                Nombre, 
                Edad, 
                Telefono, 
                TipoDeDocumento.ToString(), 
                Documento);

            Console.WriteLine(datos);
        }
    }
}
